server/src/UseCases/Common/Result/ResultBuilder.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Threading.Tasks;
+
+namespace BGS.UseCases.Common.Result;
+
+public class ResultBuilder
+{
+    public static Task<Result> BuildSucceed()
```

Comment:
Why `Task<>`? I think it's not necessary

Reply:
Removed